README.md

```diff
@@ -15,18 +15,27 @@ Command-line interface for recheck.
       --help      Display this help message.
       --version   Display version info.
 Commands:
-  help     Display help information about the specified command.
-  version  Display version info.
-  diff     Display given differences.
-  commit   Accept given differences.
-  ignore   Ignore given differences.
+  help         Display help information about the specified command.
+  version      Display version info.
+  diff         Display given differences.
+  commit       Accept given differences.
+  ignore       Ignore given differences.
+  completion   Generate and display auto completion script.
 ```
 
 
 ## Download and Installation
 
 The recheck CLI is still under development, but we will soon release a minimal working solution.
 
+## Enabling shell autocompletion
```

Comment:
Grammar Nazi: we use title-style capitalization (see "Download and Installation"), hence, it should be "Enabling Shell Auto-Completion".